libportal/remote.c

```diff
@@ -846,6 +846,16 @@ xdp_session_open_pipewire_remote (XdpSession *session)
   return g_unix_fd_list_get (fd_list, fd_out, NULL);
 }
 
+static gboolean
+is_active_remotedesktop_session (XdpSession    *session,
```

Comment:
nit: should be `is_active_remote_desktop_session()` to be consistent with other API.